modules/serverless-rn-1-26-0.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies
+//
+// * /serverless/serverless-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="serverless-rn-1-26_{context}"]
+= Release notes for Red Hat {ServerlessProductName} 1.26
+
+{ServerlessProductName} 1.26 is now available. New features, changes, and known issues that pertain to {ServerlessProductName} on {product-title} are included in this topic.
+
+[id="new-features-1.26_{context}"]
+== New features
+
+* {ServerlessProductName} now uses Knative Serving 1.5.
+* {ServerlessProductName} now uses Knative Eventing 1.5.
+* {ServerlessProductName} now uses Kourier 1.5.
+* {ServerlessProductName} now uses Knative (`kn`) CLI 1.5.
+* {ServerlessProductName} now uses Knative Kafka 1.5.
+* {ServerlessProductName} now uses Knative Operator 1.3. FIXME please confirm this needs explicit listing here
+* The `kn func` CLI plug-in now uses `func` 1.8.1.
+
+* Persistent volume claims (PVCs) are now GA. PVCs provide permanent data storage for your Knative services.
+
+* The new trigger filters feature is now available as a Technology Preview (TP). It allows users to specify a set of filter expressions, where each expression evaluates to either true or false for each event.
++
+To enable new trigger filters, add the `new-trigger-filters: enabled` entry in the section of the `KnativeEventing` type in the operator config map:
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+...
+...
+spec:
+  config:
+    features:
+      new-trigger-filters: enabled
+...
+----
+
+* Knative Operator 1.3 replaces the `v1alpha1` API for `operator.knative.dev` with the updated `v1beta1` version. FIXME let me know if "replaces" is too strong a word (if we want to say "adds" because both are available)
```

Comment:
Yeah. Probably better to say "adds" because both are now available. Thanks

Reply:
do we deprecate v1alpha1 though? (we should say so in the RN if we do)

Reply:
Yes we will remove v1alpha1 in 1.27. We should mention it. I will also update the jira.